<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="EPBYMINW2705" name="tests.task0109.Tst0901GenerateArrayAndJoin" tests="11" failures="0" timestamp="26 Jun 2017 09:41:34 GMT" time="0.020" errors="0">
  <testcase name="checkJoinTwoArrays" time="0.001" classname="tests.task0109.Tst0901GenerateArrayAndJoin"/>
  <testcase name="catchingIllegalArgExceptionNegativeInsertPosition" time="0.001" classname="tests.task0109.Tst0901GenerateArrayAndJoin"/>
  <testcase name="catchingIllegalArgException" time="0.000" classname="tests.task0109.Tst0901GenerateArrayAndJoin"/>
  <testcase name="checkLengthArrayPos" time="0.000" classname="tests.task0109.Tst0901GenerateArrayAndJoin"/>
  <testcase name="checkLengthArrayPos" time="0.000" classname="tests.task0109.Tst0901GenerateArrayAndJoin"/>
  <testcase name="checkJoinTwoArrays" time="0.000" classname="tests.task0109.Tst0901GenerateArrayAndJoin"/>
  <testcase name="checkJoinTwoArrays" time="0.001" classname="tests.task0109.Tst0901GenerateArrayAndJoin"/>
  <testcase name="checkJoinTwoArrays" time="0.000" classname="tests.task0109.Tst0901GenerateArrayAndJoin"/>
  <testcase name="catchingIllegalArgExceptionInsertPositionBiggerArrayLength" time="0.000" classname="tests.task0109.Tst0901GenerateArrayAndJoin"/>
  <testcase name="catchingIllegalArgExceptionInsertPositionBiggerArrayLength" time="0.001" classname="tests.task0109.Tst0901GenerateArrayAndJoin"/>
  <testcase name="catchingIllegalArgException" time="0.016" classname="tests.task0109.Tst0901GenerateArrayAndJoin"/>
</testsuite> <!-- tests.task0109.Tst0901GenerateArrayAndJoin -->
